test/legacy_test/test_model.py

```diff
@@ -69,14 +69,14 @@ def forward(self, x):
         return y, 0
 
 
-class ModelOutter(paddle.nn.Layer):
+class ModelOuter(paddle.nn.Layer):
     def __init__(self):
         super().__init__()
         self.module1 = ModelInner()
         self.module2 = paddle.nn.Linear(4, 5)
 
     def forward(self, x):
-        y, dummpy = self.module1(x)
+        y, dumpy = self.module1(x)
```

Comment:
```suggestion
        y, _ = self.module1(x)
```

这里应该是 `dummy`，但对于 Python 来说，未使用变量使用 `_` 是更常见的用法